src/main/kotlin/uk/gov/justice/digital/hmpps/visitallocationapi/service/NomisSyncService.kt

```diff
@@ -47,12 +47,30 @@ class NomisSyncService(
     }
 
     // If VO balance has changed, sync it
-    val wantedVoChange = calculateAmountToChange(prisonerBalance.voBalance, syncDto.oldVoBalance, syncDto.changeToVoBalance ?: 0)
-    processSync(syncDto.prisonerId, prisonerBalance.voBalance, wantedVoChange, VisitOrderType.VO, NegativeVisitOrderType.NEGATIVE_VO)
+    if (syncDto.oldVoBalance != null) {
```

Comment:
what if old balance is null but change to balance comes in? should we assume old balance to be 0 in that case?